tools/pyi/gen_pyi.py

```diff
@@ -536,6 +531,7 @@ def gen_pyi(declarations_path, out):
                      'def __init__(self, other: Tensor) -> None: ...',
                      'def __init__(self, size: {}, *, {}) -> None: ...'.format(type_to_python('IntArrayRef'), DEVICE_PARAM),
                      ],
+        'as_subclass': ["def as_subclass(self, cls) -> Tensor: ..."],
```

Comment:
Would be nice to have a more accurate type here but I won't block this PR on it.